### PR TITLE
Scrutinizer Auto-Fixes

### PR DESCRIPTION
**Pull request by scrutinizer-auto-fixer**
@padraic requested this pull request.

It consists of patches automatically generated for this project on Scrutinizer:
https://scrutinizer-ci.com/g/padraic/humbug/

### CODE REVIEW

File: src/TestSuite/Mutant/Runner.php [hunk 1]
```diff
@@ -50,6 +49,9 @@ class Runner
      */
     private $observers = [];
 
+    /**
+     * @param string $baseDirectory
```
Comment (GrahamCampbell) [inline src/TestSuite/Mutant/Runner.php]: This change looks dodgie.

File: src/Utility/CoverageData.php [hunk 2]
```diff
@@ -96,6 +99,10 @@ public function getTestClasses($file, $line)
         return $classes;
     }
 
+    /**
+     * @param string $file
+     * @param integer $line
```
Comment (GrahamCampbell) [inline src/Utility/CoverageData.php]: integer is not correct - should be int.

File: src/TestSuite/Mutant/Runner.php (hunk 1 above)
Comment (aztech-dev) [inline src/TestSuite/Mutant/Runner.php]: How so ? $baseDirectory **is** a string.... or are you referring to something else ?

Comment (oliverklee) [inline src/TestSuite/Mutant/Runner.php]: string is correct, but the @param for the other parameters still are missing.

File: src/Utility/CoverageData.php [hunk 3]
```diff
@@ -76,6 +76,9 @@ public function hasTestClasses($file, $line)
         return true;
     }
 
+    /**
+     * @param string $file
```
Comment (oliverklee) [inline src/Utility/CoverageData.php]: The annotation for the other parameter still is missing.

File: src/TestSuite/Mutant/Runner.php (hunk 1 above)
Comment (aztech-dev) [inline src/TestSuite/Mutant/Runner.php]: Ah of course :)

**Comment by padraic** [comment]
All items fixed and a few others on the PR branch ;). These auto patches are handy at times, but woefully inaccurate in some cases.

**Pull request merged**